src/features/MoviesBrowser/ListPage/PersonTile/index.js

```diff
@@ -1,12 +1,15 @@
 import { Link } from "react-router-dom";
 import { toPerson } from "../../../../core/config/routes";
+import { StyledPersonTile } from "./styled";
 
-const PersonTile = ({ person }) => {
+const PersonTile = ({ person, showJob, showCharacter }) => {
     return (
-        <div>
+        <StyledPersonTile>
             <img alt={person.name} src={`https://image.tmdb.org/t/p/w200${person.profile_path}`} />
             <Link to={toPerson(person)}>{person.name}</Link>
-        </div>
+            {showJob || <p> {person.job} </p>}
+            {showCharacter || <p> {person.character} </p>}
```

Comment:
Te 2 P do ostylowania ale to ten sam komponent

Reply:
Zrobiłam w <PersonDetails>

Reply:
Zrobiłam w <PersonDetail>